paddle/gserver/evaluators/ChunkEvaluator.cpp

```diff
@@ -243,6 +242,53 @@ class ChunkEvaluator : public Evaluator {
     if (tag == tagSingle_) return true;
     return false;
   }
+
+public:
+  // three metrics: precision, recall and F1-score
+  void getNames(std::vector<std::string>* names) {
```

Comment:
It is not a big deal I think. Making `recall`, `F1-score` just a key for value is simple for implementation.

But we could also,
```cpp
class Evaluator {
public:
  getNames(vec) {vec.extend({"recall", "precision"});}
private:
   real recall_;
   real precision;
}
```

But in view of the previous writing has been done, so be it.